lib/livebook_cli/server.ex

```diff
@@ -35,6 +35,7 @@ defmodule LivebookCLI.Server do
       --name               Set a name for the app distributed node
       --no-token           Disable token authentication, enabled by default
                            If LIVEBOOK_PASSWORD is set, it takes precedence over token auth
+      --notebook           Open browser window pointing to notebook specified
```

Comment:
```suggestion
      --open-notebook      Open browser window pointing to the given notebook
```

A bit longer, but consistent with the other ones. If someone uses this frequently, they are likely to setup an alias anyway, @josevalim any preference?

Reply:
my unsolicited opinion is +1, I was gonna propose the same. Another idea is `--open-file`. It's a separate discussion but would it make sense to have `$ livebook server --open-url https://github.com/...` too? If so perhaps `--open-file` makes a bit more sense.

Reply:
Good point, we could make `--open-notebook` accept both file path and URL, not sure if that's better or worse. Note that once we do #927 then the semantics would be different, from URL we just import, from file we actually open, but that's a sidenote.

Reply:
yeah I think we could check whether given argument starts with http(s):// (maybe we special case livebook:// too, dunno, probably not as it's supposed to be internal) and if so it's an url, otherwise it's a local file. I kinda like the more explicit `--open-file` & `--open-url` but a single option that handles both might be something more users would expect.

Reply:
Hopefully @josevalim has a strong preference :see_no_evil:

Reply:
This goes into the same discussion as #927. I think `--open-file` (or `--open-notebook`) should actually open it and not import it. So my suggestions are: `--open-file` and `--import-url`. I don't know if we have a URL for opening a notebook though. Thoughts @jonatanklosko?

Reply:
@josevalim yeah for local file we should totally open it, but I think it's fine to handle in a separate PR. As for the option name, there's actually an ambiguity here, because "open" in `--open-file`/`--open-notebook` refers to opening the browser, rather than persistence. It's similar to `--open` and `--open-new`, and that's the reasoning behind `--open-url` too. That said, `--import-url` sounds good to me, and then we are surely left with `--open-file`.

Reply:
So I'm thinking we can add `--open-file` and `--import-url` in this PR, but both of them import, then in a follow up PR we add an endpoint for opening local files and use it for `--open-file` and 927. Does that make sense?

Reply:
Hrm, to be honest, I am not sure. This sounds verbose. What if instead we make this an argument to `livebook server`? Such as `livebook server Foo.livemd`? And if a URL is given, we import?

then this PR supports only URLs and we add files later?

Reply:
Ah, we also have this option. The good thing is that we avoid the ambiguous open/import wording altogether, so it's definitely less confusing. The only question is if we would want to use argv for anything else in the future, but I cannot think of an obvious case. @wojtekmach?

Reply:
Cant think of any other use case for argv. Yet another option, to nit paint ourselves into a wall, is a more generic `—-url URL`, a file path is an url after all too. But maybe it could be mistaken for the url of the server and not of the thing we want to access. So yeah argv sounds good to me.

Reply:
Fantastic, let's do argv then!

@aar2dee2 sorry for revisiting just now :see_no_evil: To sum up, the idea is to have `livebook server [options] [.livemd url]`, so we need to get argv from option parser and if there is anything we will import the URL.

Then in a separate PR we will extend it to `[.livemd file or url]`, and when file is given we "Open" it instead of importing.